antarest/study/storage/rawstudy/model/filesystem/matrix/input_series_matrix.py

```diff
@@ -105,7 +105,14 @@ def write_dataframe(self, df: pd.DataFrame) -> None:
         if self.default_empty is not None and np.array_equal(df.to_numpy(dtype=np.float64), self.default_empty):
             self.config.path.touch(exist_ok=True)
         else:
-            df.to_csv(self.config.path, sep="\t", header=False, index=False)
+            # We're checking if the dataFrame could be represented with integer values.
+            # If so, we'll write it this way as it would be quicker and the file takes less place on the fs.
+            df_as_int = df.astype(np.int32)
```

Comment:
I think we'd better target int64 to catch larger values, the serialized form will stay the same and from my tests it does not hurt performance.